tock/hours/views.py

```diff
@@ -59,6 +60,21 @@ class SlimBulkTimecardSerializer(serializers.Serializer):
     billable = serializers.BooleanField(source='project.accounting_code.billable')
     mbnumber = serializers.CharField(source='project.mbnumber')
 
+class AdminBulkTimecardSerializer(serializers.Serializer):
+    project_name = serializers.CharField(source='project.name')
+    project_id = serializers.CharField(source='project.id')
+    employee = serializers.StringRelatedField(source='timecard.user')
+    start_date = serializers.DateField(source='timecard.reporting_period.start_date')
+    end_date = serializers.DateField(source='timecard.reporting_period.end_date')
+    hours_spent = serializers.DecimalField(max_digits=5, decimal_places=2)
+    billable = serializers.BooleanField(source='project.accounting_code.billable')
+    agency = serializers.CharField(source='project.accounting_code.agency.name')
+    flat_rate = serializers.BooleanField(source='project.accounting_code.flat_rate')
+    active = serializers.BooleanField(source='project.active')
+    mbnumber = serializers.CharField(source='project.mbnumber')
+    notes = serializers.CharField()
+    grade = serializers.CharField(source='grade.grade')
+
```

Comment:
Are our bulktimecardserializer changes meant to be in this PR? or just the parts of them that add grade?